src/app/zap-templates/zcl/data-model/chip/dishwasher-alarm.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="BITMAP32">
+    <cluster code="0x005D" />
+    <field name="InflowError"     mask="0x01" />
```

Comment:
This seems to be an issue in the spec too: bit 0 is used for the RESET feature in Alarm Base.  But this is also defining bit 0 to mean InflowError....  Is this meant to override the Alarm Base feature (and if so, what happens to the Latch attribute and Reset command for this cluster)?

Reply:
I missed this.  Seems like we should add a SHALL in the feature definition for alarm base to start with bit 1.  Or, should we reserve some number of bits in the alarm base feature map and start with bit 8 in the derived clusters?

Reply:
Reserving some bits in the base sounds like a good idea.

Reply:
Updated to remove the feature map and support the Supported attribute per PR https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7196